index.html

```diff
@@ -0,0 +1,53 @@
+<html>
+    <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>portfolio mobile</title>
+       <link rel="stylesheet" href="style.css">
+       <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    </head>
+ <body>   
+      <header class="navbar">
+        <div class="navbar__header">
+            <div class="navbar__header__content">
+              <div class="navbar__logo" href="#"> My Logo</div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As this comment was not pointed out by the previous reviewer, kindly ensure that your logo is a link (i.e. it uses a `<a>` tag). This is for three reasons: convention, semantics, and user experience(UX).